src/pages/comparison/comparison.tsx

```diff
@@ -1,5 +1,268 @@
+        elevation = oldElevation + 5;
+      }
+    }
+
+    setViewState({
+      ...viewState,
+      position: [0, 0, elevation],
+    });
+  };
+
+  const onTilesetLoad = (tileset: Tileset3D) => {
+    if (needTransitionToTileset) {
+      const { zoom, cartographicCenter } = tileset;
+      const [longitude, latitude] = cartographicCenter || [];
+      const viewport = currentViewport;
+      let pLongitue = longitude;
+      let pLatitude = latitude;
+
+      if (viewport) {
+        const { pitch, bearing } = viewState;
+        // @ts-expect-error - roperty 'fullExtent' does not exist on type 'never'.
+        const { zmin = 0 } = metadata?.layers?.[0]?.fullExtent || {};
+        /**
+         * See image in the PR https://github.com/visgl/loaders.gl/pull/2046
+         * For elevated tilesets cartographic center position of a tileset is not correct
+         * to use it as viewState position because these positions are different.
+         * We need to calculate projection of camera direction onto the ellipsoid surface.
+         * We use this projection as offset to add it to the tileset cartographic center position.
+         */
+        const projection = zmin * Math.tan((pitch * Math.PI) / 180);
+        /**
+         * Convert to world coordinate system to shift the position on some distance in meters
+         */
+        const projectedPostion = viewport.projectPosition([
+          longitude,
+          latitude,
+        ]);
+        /**
+         * Shift longitude
+         */
+        projectedPostion[0] +=
+          projection *
+          Math.sin((bearing * Math.PI) / 180) *
+          viewport.distanceScales.unitsPerMeter[0];
+        /**
+         * Shift latitude
+         */
+        projectedPostion[1] +=
+          projection *
+          Math.cos((bearing * Math.PI) / 180) *
+          viewport.distanceScales.unitsPerMeter[1];
+        /**
+         * Convert resulting coordinates to catrographic
+         */
+
+        [pLongitue, pLatitude] = viewport.unprojectPosition(projectedPostion);
+      }
+
+      const newViewState = {
+        ...viewState,
+        zoom: zoom + 2.5,
+        longitude: pLongitue,
+        latitude: pLatitude,
+      };
+
+      setViewState({
+        ...newViewState,
+        transitionDuration: TRANSITION_DURAITON,
+        transitionInterpolator: new FlyToInterpolator(),
+      });
+      setNeedTransitionToTileset(false);
+    }
+  };
+
+  const loadOptions: {
+    i3s: {
+      coordinateSystem: number;
+      token?: string;
+    };
+  } = {
+    i3s: { coordinateSystem: COORDINATE_SYSTEM.LNGLAT_OFFSETS },
+  };
+  const layers1 = [
+    new Tile3DLayer({
+      id: `tile-layer`,
+      data: "https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/SanFrancisco_3DObjects_1_7/SceneServer/layers/0",
+      loader: I3SLoader,
+      onTilesetLoad: onTilesetLoad,
+      loadOptions,
+    }),
+  ];
+  const layers2 = [
+    new Tile3DLayer({
+      id: `tile-layer`,
```

Comment:
make id of `layers1` and `layers2` different.

Reply:
done